spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -100,6 +100,7 @@ content into your application. Rather, pick only the properties that you need.
 	spring.jmx.default-domain= # JMX domain name.
 	spring.jmx.enabled=true # Expose management beans to the JMX domain.
 	spring.jmx.server=mbeanServer # MBeanServer bean name.
+	spring.jmx.jmx.unique-names=true # Set if unique runtime object names should be ensured.
```

Comment:
There is a duplicate `jmx` term there and the default value is wrong. Keys are ordered alphabetically, can you please move it at the right place?